src/hash.cpp

```diff
@@ -170,7 +174,7 @@ bcrypt bcrypt::from_hash_string(const std::string& input)
 bcrypt bcrypt::hash_pw(const std::string& password, bcrypt& salt)
 {
 	bcrypt hash;
-	if(bcrypt_hashpw(password.c_str(), salt.hash.data(), hash.hash.data()) != 0)
+	if (!php_crypt_blowfish_rn(password.c_str(), salt.hash.data(), hash.hash.data(), BCRYPT_HASHSIZE))
```

Comment:
Nitpicking: our code style mandates that there isn't space between `if` and the opening parenthesis.